src/proxy-ui-api/frontend/src/views/Clients/ServiceClients/ServiceClients.vue

```diff
@@ -1,33 +1,39 @@
 <template>
   <div>
     <div class="table-toolbar">
-      <v-text-field v-model="search" label="Search" single-line hide-details class="search-input">
+      <v-text-field v-model="search"
+                    :label="$t('serviceClients.searchPlaceHolder')"
+                    single-line
+                    hide-details
+                    class="search-input">
         <v-icon slot="append">mdi-magnify</v-icon>
       </v-text-field>
       <v-btn
         color="primary"
-        @click="addSubject"
+        @click="addServiceClient"
         outlined
         rounded
         class="ma-0 rounded-button elevation-0"
-      >{{$t('serviceClients.addSubject')}}
+      >{{$t('serviceClients.addServiceClient')}}
       </v-btn>
     </div>
 
-    <v-card flat>
-      <table class="xrd-table service-clients-table">
+    <table class="xrd-table service-clients-table">
+      <thead>
         <tr>
-          <th>{{$t('serviceClients.memberNameGroupDesc')}}</th>
+          <th>{{$t('serviceClients.name')}}</th>
           <th>{{$t('serviceClients.id')}}</th>
         </tr>
-        <template v-if="serviceClients.length > 0">
-          <tr v-for="sc in this.filteredServiceClients()">
-            <td>{{sc.subject.member_name_group_description}}</td>
-            <td>{{sc.subject.id}}</td>
+      </thead>
+      <template v-if="serviceClients.length > 0">
+        <tbody>
+          <tr v-for="sc in this.filteredServiceClients()" @click="showAccessRights(sc.id)">
```

Comment:
same 'v-bind:key'  thing in this loop also

Reply:
Fixed